app/models/embed/purl.rb

```diff
@@ -17,13 +17,14 @@ def attributes=(hash)
     attr_accessor :druid, :type, :title, :use_and_reproduction, :copyright, :contents, :collections,
                   :license, :bounding_box, :embargo_release_date, :archived_site_url, :external_url,
                   :embargoed, :citation_only, :stanford_only_unrestricted, :public, :controlled_digital_lending,
-                  :etag, :last_modified
+                  :no_download, :etag, :last_modified
 
     alias embargoed? embargoed
     alias citation_only? citation_only
     alias stanford_only_unrestricted? stanford_only_unrestricted
     alias public? public
     alias controlled_digital_lending? controlled_digital_lending
+    alias no_download? no_download
```

Comment:
Rather than adding this, can we check the file properties instead?  I find the "download" property of an object to be confusing, but the "download" property of a file is much more easily understandable.

Reply:
Yes, it is confusing.  I guess the question is, is it possible to have an object for which at the object level it says "download=none", but for specific files to have download allowed in some way?  If not (or if this situation happens but is to be treated as a metadata error and should act as **all** files are "download=none"), then looking at the object level is still appropriate.

BUT... if it is possible for an PDF object to **allow** download at the object level but **disallow** it for the first file, then this situation warrants checking the file first.  For PDFs anyway (where we only even try to show the first file in the object), this check may be useful to do.

Reply:
So I guess I would argue it may still be useful to have this method that verifies global no download at the object level available, but we need a second check (maybe only useful for PDFs) that also looks at the first file's rights...perhaps later we for other object types, we will need additional checks.  Unless we can conclusively say that the object level no download rights really have no meaning and never supercede what is set at the file level.

Reply:
Just added some logic to verify first PDF file download rule, but only for the case being handled here (PDF viewer only).

Reply:
Stacks is looking at the file level. If we don't use the file level here it may not match with what stacks thinks.  The "download" on a object is just a default setting for new files added to the object.  The canonical setting is the one on the file.

Reply:
OK, removed object level no download code, only look at the first PDF file in this case.